psm_lib.py

```
@@ -16,53 +42,332 @@ def calculate_propensity_score(df, treatment_col, covariate_cols):
+
+        # Add Threshold Lines (0.1)
+        fig.add_shape(type="line", x0=0.1, y0=0, x1=0.1, y1=1, xref='x', yref='paper',
+                      line={'color': "Gray", 'width': 1, 'dash': "dash"})
+        fig.add_annotation(x=0.1, y=1, yref='paper', text="0.1 threshold", showarrow=False, yshift=10)
+
+        fig.update_layout(
+            title="Love Plot: Standardized Mean Difference (SMD)",
+            xaxis_title="Absolute Standardized Mean Difference",
+            yaxis_title="Variables",
+            legend_title="Status",
+            margin=dict(l=0, r=0, t=40, b=0),
```

Ruff (0.14.10)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)